src/panels/UserPanelsPanel/UserPanel.tsx

```diff
@@ -0,0 +1,15 @@
+import { Flex } from '@mantine/core';
+
+interface Props {
+  src: string;
+  title: string;
+}
+
+export function UserPage({ src, title }: Props) {
```

Comment:
I would rename this file to `UserPage.tsx`, to match the component name

And also consider renaming the `UserPanelsPanel` to just `UserPanels`, to get rid of the double "Panels" in the naming. 

OBS! We need to talk about this naming scheme! Sometimes we use "Panel" in the name, and sometimes we do not (e.g. Scene, SessionRecording)